src/main/java/fi/vm/sade/valinta/kooste/valintojentoteuttaminen/HakukohteenValintatiedot.java

```diff
@@ -0,0 +1,10 @@
+package fi.vm.sade.valinta.kooste.valintojentoteuttaminen;
+
+/** Esittää eri valintojen palveluista koostettuja hakukohde-kohtaisia tietoja. */
+public class HakukohteenValintatiedot {
+  public Boolean hasValintakoe;
+
+  public HakukohteenValintatiedot(Boolean hasValintakoe) {
+    this.hasValintakoe = hasValintakoe;
+  }
+}
```

Comment:
Tää on aika hassu luokka, antaako lisäarvoa verrattuna siihen että käyttäisi suoraan boolean-arvoa tämän sijaan? Tuntuisi että hakukohteen oidin voisi ainakin pitää mukana jos halutaan tehdä näin

Reply:
Ennakoin tässä hieman, koska tähän on tulossa ainakin neljä muuta eri paikoista koostettua tietoa, joita käytetään suodatusta varten.

Reply:
Eli on ehdottomasti perusteltua tehdä luokkana, mutta lisäisin tosiaan hakukohdeOidin uutena kenttänä mukaan. Sitten dataa ei olisi pakko käsitellä aina map-muodossa (avaimena hakukohdeOid), vaan käyttäjä voi ottaa vapauksia ilman että hakukohdeOid "unohtuu"

Reply:
Lisäsin hakukohdeOidin